src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -862,6 +860,7 @@ def test_mysql_flexible_server_local_context(self, resource_group):
         self._test_flexible_server_local_context('mysql', resource_group)
 
     def _test_flexible_server_local_context(self, database_engine, resource_group):
+        self.cmd('config param-persist on')
```

Comment:
Shall we turn off `param-persist` after finishing test?

Reply:
Turned off